src/vs/workbench/contrib/search/browser/search.contribution.ts

```diff
@@ -382,6 +382,11 @@ configurationRegistry.registerConfiguration({
 			'description': nls.localize('search.experimental.quickAccess.preserveInput', "Controls whether the last typed input to Quick Search should be restored when opening it the next time."),
 			'default': false
 		},
+		'search.quickOpen.seedStringFromSelection': {
```

Comment:
We should move this to be next to the other quickOpen settings for consistency

Reply:
Done!